src/mpl2/src/object.h

```diff
@@ -778,17 +778,18 @@ struct Rect
     f_y = f_y_;
   }
 
-  bool isValid() const
+  bool isValid() const { return (lx < ux) && (ly < uy); }
```

Comment:
Is zero width or height invalid?

Reply:
Considering it's used for guidance and fence regions only (for now), I think it's reasonable to make a zero height/width region invalid.